libtcmu.c

```diff
@@ -835,6 +835,11 @@ void tcmu_dev_set_private(struct tcmu_device *dev, void *private)
 	dev->hm_private = private;
 }
 
+const char *tcmu_dev_get_uio_name(struct tcmu_device *dev)
+{
+	return dev->tcm_dev_name;
```

Comment:
tcm_dev_name is the LIO name, but the function is named after the uio name. You probably wanted:

return dev->dev_name;

Reply:
Yeah, it is. Checked it locally, the diff code is still in the stash.
Will push it again.